### ISSUE TITLE
macOS style rounded corners

### ISSUE THREAD

**Issue by net** [lwouis/alt-tab-macos#21]
Rounded corners would help the switcher blend into the OS.

**Comment by pcr910303** [comment]
Yeah, rounded corners and less prominent (grey instead of white) text colors will definitely make this more mac-esque instead of looking Window-ey.

**Comment by lwouis** [comment]
I actually prefer Windows style, but I understand preferences may vary. I think this should be a preference supported in some UI. There is already a ticket about that (https://github.com/lwouis/alt-tab-macos/issues/8), but I'll keep this separate one open to not have a god-ticket with a million features.

**Comment by lwouis** [comment]
[Release 1.6.0](https://github.com/lwouis/alt-tab-macos/releases/tag/v1.6.0) switches to a macOS theme. I kept the previous Windows 10 theme as a preference.

Hope you like it!

Note that I made the background a bit darker than the native app switcher because the text is difficult to read otherwise.

**Comment by Lexxie9952** [comment]
What would be super nice is the best of both. Rounded corners à la Mac but highlighted white frame à la Windows. I dislike the Windows theme but feel forced to use it because the Mac theme provides not enough contrast. This creates obstacle friction for any serious Power-User to use the more aesthetically pleasing Mac theme.

**Comment by lwouis** [comment]
Hi @Lexxie9952,

The next release will have #351. It will feature a better UI for the switcher and a High Visibility mode which will combine macOS aesthetics with Windows 11 usability, as you wished here.

Thank you
